.github/workflows/mac.yml

```
@@ -0,0 +1,122 @@
+name: MACOS
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - develop
+      - master
+  push:
+    branches:
+      - develop
+      - master
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  main:
+    runs-on: macos-10.15
+    strategy:
+      matrix:
+        include:
+          - cubit: 17.1.0
+            URL: MAC_CUBIT_17_1_0
+            URL_SDK: MAC_CUBIT_SDK_17_1_0
+            CUBIT_BASE_NAME: Trelis-17.1
+            
+          - cubit: 2020.2
+            URL: MAC_CUBIT_2020_2
+            URL_SDK: MAC_CUBIT_SDK_2020_2
+            CUBIT_BASE_NAME: Coreform-Cubit-2020.2
+
+    name: 'Trelis/Cubit Svalinn Plugin ${{ matrix.cubit }} Build for Macos'
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Environment Variables
+        shell: bash -l {0}
+        run: |
+          echo "CURRENT=$(pwd)" >> $GITHUB_ENV
+          echo "SCRIPTPATH=$GITHUB_WORKSPACE" >> $GITHUB_ENV
+          echo "PLUGIN_ABS_PATH=$GITHUB_WORKSPACE/.." >> $GITHUB_ENV
+          echo "FOLDER_PKG=$GITHUB_WORKSPACE/../pkg" >> $GITHUB_ENV
+          echo "CUBIT_PATH=/Applications/${{ matrix.CUBIT_BASE_NAME }}.app/Contents" >> $GITHUB_ENV
+          if [ "${{ matrix.cubit }}" = "17.1.0" ]; then
+            echo "CMAKE_ADDITIONAL_FLAGS=-DCMAKE_CXX_FLAGS=-D_GLIBCXX_USE_CXX11_ABI=0" >> $GITHUB_ENV
+            echo "TRELIS_PKG=Trelis-17.1.0-Mac64.dmg" >> $GITHUB_ENV
+            echo "TRELIS_SDK_PKG=Trelis-SDK-17.1.0-Mac64.tar.gz" >> $GITHUB_ENV
```

Should these be changed to CUBIT as well?
```suggestion
            echo "CUBIT_PKG=Trelis-17.1.0-Mac64.dmg" >> $GITHUB_ENV
            echo "CUBIT_SDK_PKG=Trelis-SDK-17.1.0-Mac64.tar.gz" >> $GITHUB_ENV
```